var/spack/repos/builtin/packages/turbine/package.py

```diff
@@ -16,23 +16,27 @@ class Turbine(AutotoolsPackage):
     configure_directory = 'turbine/code'
 
     version('develop', branch='master')
-    version('1.2.3', 'f2e393c292c4248b4e77a19f8272ae88')
+    version('1.2.3', '028f6f4f5041e5ffbbf3ab8008e3b781')
     version('1.2.1', 'c8976b22849aafe02a8fb4259dfed434')
     version('1.1.0', '9a347cf16df02707cb529f96c265a082')
 
     variant('python', default=False,
             description='Enable calling python')
+    variant('python3', default=False,
+            description='Enable calling python3')
     variant('r', default=False,
             description='Enable calling R')
     variant('hdf5', default=False,
             description='Enable HDF5 support')
+    depends_on('adlbx@develop', when='@develop')
     depends_on('adlbx@:0.8.0', when='@:1.1.0')
     depends_on('adlbx', when='@1.2.1:')
     depends_on('adlbx')
     depends_on('tcl', type=('build', 'run'))
     depends_on('zsh', type=('build', 'run'))
     depends_on('swig', type='build')
     depends_on('python', when='+python')
+    depends_on('python@3', when='+python3')
```

Comment:
Spack doesn't always handle shortened versions well. In this case, it might try to install `python@3` instead of "the latest version of Python 3. Also, I wouldn't add a `python3` variant. If users want to use `python3`, they can do:
```console
$ spack install turbine+python ^python@3:
```

Reply:
Removed in the latest PR commits.